index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=]. Examples include
+        ISO mobile driver's licenses [[ISO18013-5]] and W3C [=verifiable
+        credentials=] [[VC-DATA-MODEL-2.0]].
+      </dd>
+      <dt>
+        <dfn>Credential provider</dfn>
+      </dt>
+      <dd>
+        Software that provides a user interface for querying, selecting, and
+        establishing consent for the [=presentation=] of a [=digital
+        credential=]. Examples include a digital wallet, or a password manager,
+        that manages various
+        <a href="#credential-type-examples">credential types</a>.
       </dd>
       <dt>
-        <dfn>Identity credential provider</dfn>
+        <dfn>Presentation request</dfn>
```

Comment:
Maybe we don't need this... "exchange protocol" could cover both request and response. WDYT?

Reply:
+1 to removing presentation request.

If the intention is to separate three things you list, manu, i think it is cleaner to use terms `query language`, `credential format`, `exchange protocol` respectively.

Reply:
I'd be hesitant to say "query language", as it seems to imply something like SQL. We are just passing a structure around, so, maybe, "query structure", or, just "query" (remember that in context it will be `[=digital credential/query=]`).

Reply:
I've made an attempt at using `query` in cd3dc633f50cf30e27de9474e919ea354a822de1.

FWIW, "query" is what the Verifiable Presentation Request specification uses: https://w3c-ccg.github.io/vp-request-spec/#query-by-example